### ISSUE TITLE
A resurrection case in Russia?

### ISSUE THREAD

**Issue by alanphilipm** [CSSEGISandData/COVID-19#1331]
on 21-03-20 daily case was reported 1 death which was then 0 on 22/03/20.

**Comment by yetzt** [comment]
in soviet russia corona is infected by you

**Comment by mrt0rtikize** [comment]
according to who.org 22/03/20 0 dead is correct
https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200322-sitrep-62-covid-19.pdf?sfvrsn=f7764c46_2

so it's resurrection or maybe 1 dead was just kidding)

**Comment by alanphilipm** [comment]
> according to who.org 22/03/20 0 dead is correct
> https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200322-sitrep-62-covid-19.pdf?sfvrsn=f7764c46_2
> 
> so it's resurrection or maybe 1 dead was just kidding)

I have just some concerns about the tidiness of the data provided here.

**Comment by merrik** [comment]
This is the first victim, she died having a positive test but Russian authorities claim she died from a stroke after an operation not related to covid. She was counted for some time and then kind of removed. I don't think this data should follow Russian authorities that tried to postpone deaths for as long as possible, she was confirmed positive. 

State agency confirms:
https://ria.ru/20200319/1568843637.html

**Comment by yetzt** [comment]
yeah, according to guidelines, deaths should not count *died from covid* but *died with covid*, it's about epidemology not about mortality.

**Comment by grwlf** [comment]
According to Yandex data, there was 1 recovered case disappeared in Sverdlovsk oblast between 29 and 30 of March.
For reference, here is an unofficial clone with details on russia (data by Yandex) https://github.com/grwlf/COVID-19_plus_Russia
